build.py

```diff
@@ -259,10 +259,26 @@ def configureQtBase(self):
             "-nomake", "examples",
             "-nomake", "tools",
             "-nomake", "tests",
-            "-no-qml-debug",
+            "-no-audio-backend",
             "-no-dbus",
+            "-no-gstreamer",
+            "-no-icu",
```

Comment:
> It's still a valid configuration if Qt is built with -no-icu, and WebKit with ICU.

I have the same feeling, it would be confusing if they are not using ICU. @glensc Let's not include `no-icu` in the build flags.